File: problemas/iniciante/2879/2879.clj
(let [n (read-string (read-line))
      vitorias (atom 0)]
  (doseq [_ (range 0 n)]
    (let [x (read-string (read-line))]
      (if (not= x 1) (swap! vitorias inc) ())))
  (printf "%d\n" @vitorias))

File: problemas/iniciante/2879/2879.cpp
#include <iostream>

using namespace std;

int main(){
    int T, N, vitorias;

    cin >> T;

    vitorias = 0;
    for(int i = 0; i < T; ++i){
        cin >> N;

        if (N != 1){
            vitorias += 1;
        }
    }

    cout << vitorias << endl;

    return 0;
}